simcon_project/users/views/create_assignment.py

```diff
@@ -0,0 +1,156 @@
+from django.shortcuts import render
+from users.models import SubjectLabel, Assignment, Student, Researcher
+from conversation_templates.models import ConversationTemplate
+from django.views.generic import TemplateView
+from django.core import serializers
+from django.http import HttpResponse
+from django.core.mail import send_mail
+from apscheduler.schedulers.background import BackgroundScheduler
```

Comment:
i looked into this 
https://docs.djangoproject.com/en/2.2/howto/custom-management-commands/#howto-custom-management-commands
https://stackoverflow.com/questions/573618/set-up-a-scheduled-job
and this seems the way to do it!